_specs/ecip-1084.md

```diff
@@ -0,0 +1,139 @@
+---
+lang: en
+ecip: 1084
+title: Hard-cap gas-limit in protocol
+author: Bob Summerwill (@bobsummerwill)
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-01-17
+license: Apache-2.0
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/260
+---
+
+## Abstract
+
+A proposal to place a hard cap on the gaslimit for the ETC mainnet as an in-protocol
+consensus rule. From the block that this ECIP was activated, gaslimit would follow a
+curve defined in this proposal rather than being subject to miner voting.
+
+## Rationale
+
+There is nothing more important to the ETC network than security and decentralization.
+Both of those desired characteristics have already been heavily compromised in ETC
+because of the malincentives which ETC inherited from ETH in the form of the miner
+gaslimit voting mechanism.
+
+The gaslimit has been raised for the ETH mainnet first to 8M and then to 10M gas,
+despite it being obvious that the client implementations cannot adequately cope with
+the state trie growth that such a high gaslimit has created.  ETH has been
+"overclocked" and smoke is coming out of the engine.  This long crisis was one of the
```

Comment:
This is not obvious to me. My client seems to run fine on ETH, ETC, and test networks. Can you provide benchmarks here?